src/main/python/yadtminion/__init__.py

```diff
@@ -275,36 +300,45 @@ def __init__(self, only_config=False):
                                               'yumdeps',
                                               'service_defs',
                                               'artefacts_filter']]
+        sys.stdout = old_stdout
+        del old_stdout
```

Comment:
To be sure that you really restore the old_stdout even if an exception occurs, you should do
```
try:
    ....
finally:
    sys.stdout = old_stdout
```
Also, I'm not sure if the `del` has any effect.